client/jetpack-connect/signup.js

```diff
@@ -83,11 +95,47 @@ export class JetpackSignup extends Component {
 		} );
 	}
 
+	componentDidUpdate( prevProps ) {
+		const { requestError } = this.props;
+
+		if ( prevProps.requestError || ! requestError ) {
+			return;
+		}
+
+		if (
+			this.getWooDnaConfig() &&
+			'usernameOrEmail' === requestError.field &&
+			'unknown_user' === requestError.code
+		) {
+			this.showWooDnaSignupView();
```

Comment:
How did the original flow handle getting an unknown using when logging in?

Reply:
It just shows an error:
<img width="674" alt="Screenshot 2020-05-31 at 12 55 02" src="https://user-images.githubusercontent.com/1715800/83351756-a2a14800-a33e-11ea-82d6-7c064686dfc3.png">